lib/house.rb

```diff
@@ -0,0 +1,14 @@
+require 'minitest'
+require './lib/room'
```

Comment:
We never need to require minitest in a class file. We only do that in test files.  The room file also does not need to be required in our `House` class file, because we don't need to instantiate a `Room` object in this file. Nothing in the interaction pattern asks us to do that, BUT we can still pass in `Room` objects that we create in our test via a method we create in our `House` file.